.gitignore

```diff
@@ -0,0 +1,17 @@
+# Mac stuff
+.DS_Store
+
+# Compiled Object files
+*.slo
+*.lo
+*.o
+
+# Compiled Dynamic libraries
+*.so
+*.dylib
+
+# Compiled Static libraries
+*.lai
+*.la
+*.a
+
```

Comment:
I don't think that it does (though I still don't understand why it would be necessary -- none of these file types are created by an Arduino build, as the Arduino app puts them into temp directories elsewhere on the system...